backend/src/routes/v0/settings.py

```
@@ -47,21 +47,21 @@
 @router.get("/settings", response_model=SettingsListResponse)
 async def list_settings(
     user: User = Depends(verify_credentials),
-    db: Session = Depends(get_db)
+    db: Session = Depends(get_async_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)